### DISCUSSION TITLE
conflict between Copilot Chat and Visual Assist extensions

### DISCUSSION THREAD

**Discussion by croguet2** [community/community#101807] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I've been using the copilot extensions with VS 2022 on PC in my organization.  We found the following issue for many users :

prerequisite : have github copilot extension + github copilot chat extension + "Visual Assist" extension installed; open a solution with VS2022 
symptom : go to some source, try to open the inline chat (Alt+: or contextual menu)  :  the inline copilot chat input window then pops up, but it doesn't react to keyboard input; there is no blinking cursor inside it.
found workaround : go to the VS menu Extensions -> Manage Extensions ... -> Installed      and "Disable" the Visual Assist extension; restart VS; the chat window now has a blinking cursor and receives input

other symptom : in the free chat window (not the inline window) copilot receives input and answers questions but is unable to fetch the context from source files; well sometimes it works and sometimes not. 
the same workaround as above also fixes this issue.

This is really annoying because we use the VA extension all the time to navigate our huge code bases. This is preventing users from trying GH copilot. 

Thanks in advance !

**Comment by ayim** [comment]
I'm going to file a tracking issue here: https://developercommunity.visualstudio.com/VisualStudio/report?port=1025&fsid=6a75721a-5f18-4846-8c77-cf0ee21b119d

This creates an internal ticket that the developers would triage and work on. Please stay in touch with us there.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
gh: ayim | aayim@microsoft.com

**Reply by ayim** [reply, marked as answer]
Per user: It was fixed by the VA team in build Build 2024.2 , see https://docs.wholetomato.com/default.asp?W404
